app/0.11.x/lua-reference/modules/kong.tools.responses.html

```diff
@@ -182,7 +182,7 @@ <h4><a name="status_codes">status_codes</a></h4>
     Define the most common HTTP status codes for sugar methods.
  Each of those status will generate a helper method (sugar)
  attached to this exported module prefixed with <code>send_</code>.
- Final signature of those methods will be <code>send_&lt;status_code_key&gt;(message, headers)</code>. See <a href="../modules/kong.tools.responses.html#send">send</a> for more details on those parameters.
+ Final signature of those methods will be <code>send_&lt;status_code_key&gt;(message, headers)</code>. See <a href="send">send</a> for more details on those parameters.
```

Comment:
Is this missing a # in the anchor tag?

Reply:
Ture, that needs a # indeed.

Reply:
Potentially this issue exists elsewhere - can we presume you'll find and fix all occurrences of this @moaiandin ?